app/code/Magento/Eav/Setup/EavSetup.php

```diff
@@ -567,7 +577,7 @@ public function addAttributeGroup($entityTypeId, $setId, $name, $sortOrder = nul
             if (empty($data['attribute_group_code'])) {
                 if (empty($attributeGroupCode)) {
                     // in the following code md5 is not used for security purposes
-                    $attributeGroupCode = md5($name);
+                    $attributeGroupCode = sha1($name);
```

Comment:
Nope, we don't need it here. It's a stupid rule, just suppress.

Reply:
yep, i'm currently do this

Reply:
@orlangur done